app/code/core/Mage/Core/etc/system.xml

```diff
@@ -1055,9 +1054,17 @@
                             <show_in_website>0</show_in_website>
                             <show_in_store>0</show_in_store>
                         </loading_timeout>
+                        <copy_path translate="label">
+                            <label>Allow to copy/paste roles and configuration path</label>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
```

Comment:
Instead of yes/no:
1. allow in developer mode  only (enabled for developer mode)
2. always allow (enabled)
3. always do not allow (disabled)